components/userPages/Login/UserInput.js

```diff
@@ -28,5 +30,6 @@ export default function UserInput({
 UserInput.defaultProps = {
   _label: "Label:",
   _type: "text",
+  _placeholder: "  Skriv här",
```

Comment:
Instead of using `defaultProps`, you can assign default values to the properties as they are being accepted by the component function, i.e.: 

```js
export default function UserInput({
  _label = 'Label:',
  _type = 'text',
  _placeholder: 'Skriv här',
  _ref, // no default
  // etc...
}) {
  // component code here
}
```